### PR TITLE
Delete datasets

### PR DESCRIPTION
**Pull request by sakeerthy**

### CODE REVIEW

File: c/datasetService.c [hunk 1]
```diff
@@ -129,6 +153,15 @@ static int serveVSAMDatasetContents(HttpService *service, HttpResponse *response
...
     updateVSAMDataset(response, filename, cache->acbTable, TRUE);
   }
+  else if (!strcmp(request->method, methodDELETE)) {
+    char *l1 = stringListPrint(request->parsedFile, 1, 1, "/", 0);
+    char *percentDecoded = cleanURLParamValue(response->slh, l1);
+    char *filenamep1 = stringConcatenate(response->slh, "//'", percentDecoded);
+    char *filename = stringConcatenate(response->slh, filenamep1, "'");
+    zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_INFO, "Deleting if exists: %s\n", filename);
+    fflush(stdout);
+    deleteVSAMDataset(response, filename);
```
Comment (sakeerthy) [inline c/datasetService.c]: datasetname

**Comment by ifakhrutdinov** [comment]
Why are there so many commits in this pull-request?

File: dco-signoffs/Matt Hogstrom-zss.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+I, Matt Hogstrom hereby sign-off-by all of my past commits to this repo subject to the Developer Certificate of Origin (DCO), Version 1.1. In the past I have used emails: matt@hogstrom.org
+
```
Comment (DivergentEuropeans) [inline dco-signoffs/Matt Hogstrom-zss.txt]: This wasn't intentional was it? Your branch may have diverged too heavily from staging

**Comment by 1000TurquoisePogs** [comment]
Suneeth said I should delete this, to be replaced with #104

**Pull request closed without merging**